Devon333/mern-project
# Dev

# Description


created separate pages, login, and register features
## Related Issues



## How to test
enter paths without logging in, login, register, navigate through app check app for missing data fields.
